src/api/cron.json

```
@@ -2,6 +2,9 @@
   "jobs": [
     {
       "command": "*/10 * * * * python -m qualicharge refresh-static --concurrently"
+    },
+    {
+      "command": "12 8 * * * dbclient-fetcher psql && psql $SCALINGO_POSTGRESQL_URL -f scripts/clean-orphans.sql"
```

We should check the `SCALINGO_POSTGRESQL_URL` value first.

```suggestion
      "command": "12 3 * * * dbclient-fetcher psql && psql $SCALINGO_POSTGRESQL_URL -f scripts/clean-orphans.sql"
```